### PR TITLE
You can now stick patches onto yourself without any delay

### PR DESCRIPTION
**Pull request by ghost**
:cl: Donald Trump
fix: You can now apply patches to yourself without any delay, who the fuck ever added this?
/:cl:

### CODE REVIEW

**Comment by tortellinitony** [comment]
the else clause works?

**Comment by KittyKathy** [comment]
Can you apply patches and bruise packs/ointment through hardsuits? We could do that pre rebase I think even though it did not make sense. Also tg randomly making patches have a max chemical limit of 40 instead of the standard 50 that pills have...like why just nerf the chemicals if brute patches were op.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -50,14 +50,6 @@
 	if(!canconsume(M, user))
 		return 0
 
-	if(M == user)
```
Comment (j-mie) [inline code/modules/reagents/reagent_containers/pill.dm]: I think you still need this?

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -50,14 +50,6 @@
 	if(!canconsume(M, user))
 		return 0
 
-	if(M == user)
-		M.visible_message("<span class='notice'>[user] attempts to [apply_method] [src].</span>")
-		if(self_delay)
-			if(!do_mob(user, M, self_delay))
```
Comment (j-mie) [inline code/modules/reagents/reagent_containers/pill.dm]: Surely you just remove self_delay?

Comment (ghost) [reply code/modules/reagents/reagent_containers/pill.dm]: Yeah i could've done that instead

**Comment by Sarcalogo** [comment]
>Can you apply patches and bruise packs/ointment through hardsuits?

Please keep this feature. It was really stupid in the old code.

**Comment by KittyKathy** [comment]
Why though how are you supposed to heal in a hardsuit/any similar suit?Do you really expect people to take their suit off in space? as that is the only place where you need a hardsuit on.

**Comment by Sarcalogo** [comment]
You don't take it off in space, that's the point. It makes people in hardsuits more vulnerable.

**Comment by KittyKathy** [comment]
Making people in hardsuits more vulnerable...someone call the balancing police NUKE OPS NERF, as if the mode is not bad enough with hippie players as teammates. Also that's not a real reason care to explain why they need to be more vulnerable?

**Comment by Sarcalogo** [comment]
Hardsuit wizard.

**Comment by ghost** [comment]
+1 fuck /tg/ patches

**Review by j-mie — APPROVED** [review]

**Pull request merged**